src/jdk.compiler/share/classes/com/sun/tools/javac/model/JavacElements.java

```diff
@@ -85,6 +85,7 @@
  * risk.  This code and its internal interfaces are subject to change
  * or deletion without notice.</b></p>
  */
+@SuppressWarnings("preview")
```

Comment:
isn't it better to apply the annotation to the method below?

Reply:
Yeap, this is what I thought at first but I was getting  "warnings found and -Werror specified". So compilation was failing.

Reply:
I think the suppress warnings annotation can be deleted now that the isUnnamed method has been removed.